src/mhd2xds.coffee

```diff
@@ -22,6 +39,13 @@ exports.mhd1Metadata2Xds = (metadata) ->
     seconds = '0' + seconds
   now = "#{year}#{month}#{day}#{hours}#{minutes}#{seconds}"
 
+  if metadata.documentEntry.patientId.indexOf '&' is -1 and metadata.documentEntry.patientId.indexOf 'ZAF' isnt -1
+    sourceId = metadata.documentEntry.patientId.replace 'ZAF', 'ZAF&0.0.0&ISO'
```

Comment:
We may need to support more than just ZAF as passport numbers are also catered for. In theory any ISO 3166-1 alpha-3 code is supported but I think only a few countries are used in production (ZWE and MOZ I think). You'll have to check with @armageddon